parm/aero/variational/3dvar_gfs_aero_bump.yaml

```diff
@@ -0,0 +1,42 @@
+cost function:
+  cost type: 3D-Var
+  window begin: '$(AERO_WINDOW_BEGIN)'
+  window length: $(AERO_WINDOW_LENGTH)
+  analysis variables: &3dvars [ua,va,t,ps,sphum,ice_wat,liq_wat,o3mr]
+  geometry: $(GEOM_BKG)
+  background:
+    datapath: $(BKG_DIR)
+    filetype: fms restart
+    datetime: $(BKG_ISOTIME)
+    filename_core: $(BKG_YYYYmmddHHMMSS).fv_core.res.nc
+    filename_trcr: $(BKG_YYYYmmddHHMMSS).fv_tracer.res.nc
+    filename_cplr: $(BKG_YYYYmmddHHMMSS).coupler.res
+    state variables: [t,delp,sphum,sulf,bc1,bc2,oc1,oc2,
+                      dust1,dust2,dust3,dust4,dust5,
+                      seas1,seas2,seas3,seas4] 
+  background error: $<< $(BERROR_YAML)
+  observations: $<< $(OBS_LIST)
+variational:
+  minimizer:
+    algorithm: DRIPCG
+  iterations:
+  - ninner: 35 
+    gradient norm reduction: 1e-10
+    test: on
+    geometry: $(GEOM_ANL)
+    diagnostics:
+      departures: bkgmob
+  - ninner: 35 
+    gradient norm reduction: 1e-10
+    test: on
+    geometry: $(GEOM_ANL)
+    diagnostics:
+      departures: bkgmob1
+final:
+  diagnostics:
+    departures: anlmob
+  increment:
+    output:
```

Comment:
FV3 cannot currently read aerosol increments. We should for our prototype testing instead write out the analysis in FMS restart format.

Reply:
I don't think this is quite right, it needs to match your 3dvar YAML you've been running with

Reply:
```
final:
  diagnostics:
    departures: anlmob
output:
  filetype: fms restart
  datapath: $(ANL_DIR)
  prefix: aeroanl
```
something like this?

Reply:
yes. I was trying to figure out what $(GEOM_ANL) includes (like nx, ny, fieldsets etc) since these are not in the template. Shouldn't GEOM_ANL be included here as well?

Reply:
you only need to specify a GEOM_ANL if the geometry for the analysis differs from the background geometry. For example, if we have C384 model but C192 DA. But we want the 'analysis' written out at the full resolution, so that is the default. If we specified the analysis geometry here, it would write out C192 RESTART files, which the model wouldn't be able to use.